### PR TITLE
587-new-resource-search-bar-UI-implementation

### PR DESCRIPTION
**Pull request by Xerxi88**

### CODE REVIEW

**Review by JuditKaramazov — DISMISSED** [review]

File: front/src/locales/cat.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
   "No es posible borrar el tema.": "No és possible esborrar el tema.",
 
   "Filtros": "Filtres",
-  "Recursos de (category)": "Recursos de {{name}}",
+  "Recursos de": "Recursos de",
```
Comment (NataliaR-BCN) [inline front/src/locales/cat.json:74]: Reverse this change

File: front/src/locales/en.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
   "No es posible borrar el tema.": "The topic cannot be deleted.",
 
   "Filtros": "Filters",
-  "Recursos de (category)": "{{name}} resources",
+  "Recursos de": "Resources of",
   "Buscar recurso": "Find resource",
```
Comment (NataliaR-BCN) [inline front/src/locales/en.json:75]: Reverse this change

File: front/src/locales/es.json [hunk 3]
```diff
@@ -71,7 +71,7 @@
   "No es posible borrar el tema.": "No es posible borrar el tema.",
 
   "Filtros": "Filtros",
-  "Recursos de (category)": "Recursos de {{name}}",
+  "Recursos de": "Recursos de",
```
Comment (NataliaR-BCN) [inline front/src/locales/es.json:74]: Reverse this change

File: front/src/__tests__/pages/Category.test.tsx [hunk 4]
```diff
@@ -48,102 +46,91 @@ describe.skip('Resource', () => {
...
-
 it('renders correctly', () => {
   render(<Category />)
-
   expect(screen.getByTestId('filters-container')).toBeInTheDocument()
   expect(screen.getByTestId('types-filter')).toBeInTheDocument()
   expect(screen.getByTestId('status-filter')).toBeInTheDocument()
   expect(screen.getByTestId('resource-list')).toBeInTheDocument()
-  expect(screen.getByText('Recursos de React')).toBeInTheDocument()
   expect(screen.getByText('Els meus recursos')).toBeInTheDocument()
```
Comment (NataliaR-BCN) [inline front/src/__tests__/pages/Category.test.tsx:55]: Undo this change

**Review by NataliaR-BCN — CHANGES_REQUESTED** [review]
Please, don't remove the ability to translate correctly the text and to test it works

File: front/src/pages/Category.tsx [hunk 5]
```diff
@@ -351,11 +390,12 @@ const CloseFilterButton = styled(Button)`
   }
 `
 
+type SortOrder = 'asc' | 'desc'
```
Comment (NataliaR-BCN) [inline front/src/pages/Category.tsx:393]: This is old code!

**Review by JuditKaramazov — DISMISSED** [review]

**Review by NataliaR-BCN — CHANGES_REQUESTED** [review]

File: front/src/pages/Category.tsx (hunk 5 above)
Comment (JuditKaramazov) [reply front/src/pages/Category.tsx:393]: Here’s the new version: [Category.tsx](https://github.com/IT-Academy-BCN/ita-wiki/blob/main/front/src/pages/Category.tsx)

File: front/CHANGELOG.md [hunk 6]
```diff
@@ -47,8 +52,19 @@ All notable changes to this project will be documented in this file.
 
 - My Resources widget in Category page
 
+## [0.3.0] - 2023-11-16
```
Comment (kevinmamaqi) [inline front/CHANGELOG.md:55]: Eliminar

File: front/CHANGELOG.md [hunk 7]
```diff
@@ -47,8 +52,19 @@ All notable changes to this project will be documented in this file.
...
+
+## Added
```
Comment (kevinmamaqi) [inline front/CHANGELOG.md:57]: Eliminar

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

File: front/src/__tests__/pages/Category.test.tsx [hunk 8]
```diff
@@ -48,102 +46,92 @@ describe.skip('Resource', () => {
...
-
-  fireEvent.click(screen.getByRole('button', { name: '+ Crear nou recurs' }))
+  render(<Category />);
+  fireEvent.click(screen.getByTestId('new-resource-text'))
   const modalTitle = screen.getByRole('heading', {
-    name: /nuevo recurso/i,
-  })
-  expect(modalTitle).toBeInTheDocument()
-  fireEvent.keyDown(document, { key: 'Escape' })
+    name: /Nuevo Recurso/i,
```
Comment (NataliaR-BCN) [inline front/src/__tests__/pages/Category.test.tsx:118]: This text is not translated?

**Review by NataliaR-BCN — APPROVED** [review]

**Review by JuditKaramazov — APPROVED** [review]

**Review by missvader — APPROVED** [review]

**Review by missvader — APPROVED** [review]

**Review by kevinmamaqi — APPROVED** [review]
